src/user/screens/profile.screen.js

```diff
@@ -147,7 +168,7 @@ class Profile extends Component {
               </SectionList>
             </View>}
         </ParallaxScroll>
```

Comment:
`<ParallaxScroll>` and its contents should be indented in one level due to the addition of the `<ScrollView>`.